docs/source/whatsnew/version8.rst

```diff
@@ -305,12 +305,8 @@ For more information please see the following unit test : ``extensions/tests/tes
 Auto formatting with black in the CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If ``black`` is installed in the same environment as IPython, terminal IPython
-will now *by default*  reformat the code in the CLI when possible. You can
-disable this with ``--TerminalInteractiveShell.autoformatter=None``.
-
-This feature was present in 7.x but disabled by default.
-
+In 8.0, input was automatically reformatted with Black.
+This feature has been reverted.
```

Comment:
I would at least include how to reenable it, `--TerminalInteractiveShell.autoformatter=black`, I believe. I was very much not aware this was possible in version 7.

FYI, one of the main problems is "If ``black`` is installed in the same environment as IPython" indicates it is not a hard requirement, but it was. If black had been opt in by installing in the same environment, maybe this wouldn't have been as bad (minus a couple of actual issues like trailing semicolon stripping). Maybe I have more faith in people controlling their environments than called for, though.

```suggestion
In 8.0, input was automatically reformatted with Black.
This feature has been reverted as a default; it can be manually enabled with ``--TerminalInteractiveShell.autoformatter=black``.
```

(Something like that)

Reply:
I agree that this could be useful but I'll let @Carreau decide what course of action to take first.